src/scripts/devhub.zig

```diff
@@ -94,17 +130,33 @@ fn upload_run(shell: *Shell, run: Run) !void {
     try shell.exec("git push", .{});
 }
 
-fn get_measurement(
-    benchmark_stdout: []const u8,
-    comptime label: []const u8,
-    comptime unit: []const u8,
-) !u64 {
-    errdefer {
-        std.log.err("can't extract '" ++ label ++ "' measurement", .{});
-    }
-
-    var cut = stdx.cut(benchmark_stdout, label ++ " = ") orelse return error.BadMeasurement;
-    cut = stdx.cut(cut.suffix, " " ++ unit) orelse return error.BadMeasurement;
+const NyrkioRun = struct {
+    const Metric = struct {
+        name: []const u8,
+        unit: []const u8,
+        value: u64,
+    };
+    timestamp: u64,
+    metrics: []const Metric,
+    attributes: struct {
+        git_repo: []const u8,
+        branch: []const u8,
+        git_commit: []const u8,
+    },
+};
 
-    return try std.fmt.parseInt(u64, cut.prefix, 10);
+fn upload_nyrkio(shell: *Shell, run: NyrkioRun) !void {
+    const token = try shell.env_get("NYRKIO_TOKEN");
+    const payload = try std.json.stringifyAlloc(shell.arena.allocator(), [_]NyrkioRun{run}, .{});
+    try shell.exec(
```

Comment:
No love for `std.http.Client`? :wink:

Reply:
Wait, we _already_ have http.Client? Didn't realize that, thought that's the next release. But, given https://github.com/ziglang/zig/pull/18955, I'd probably let it bake a bit more until we depend on it